lib/lighting/src/lighting.cpp

```diff
@@ -0,0 +1,123 @@
+#include <lighting.h>
+
+FASTLED_USING_NAMESPACE
+
+// FastLED "100-lines-of-code" demo reel, showing just a few 
+// of the kinds of animation patterns you can quickly and easily 
+// compose using FastLED.  
+//
+// This example also shows one easy way to define multiple 
+// animations patterns and have them automatically rotate.
+//
+// -Mark Kriegsman, December 2014
+
+#if defined(FASTLED_VERSION) && (FASTLED_VERSION < 3001000)
+#warning "Requires FastLED 3.1 or later; check github for latest code."
+#endif
+
+#define DATA_PIN    NEO_PIXEL_PIN
+#define LED_TYPE    WS2811
+#define COLOR_ORDER GRB
+#define NUM_LEDS    NEO_PIXEL_COUNT
+CRGB leds[NUM_LEDS];
+
+#define BRIGHTNESS          96
+#define FRAMES_PER_SECOND  120
+
+void lighting_setup() {
```

Comment:
could probably convert everything here to a class when we start implementing the real patterns that way we dont have to worry about namespacing

i could imagine something like this in main.cpp:

```cpp
Lighting l;

void setup() {
 l.init(lightingPin, numPixels);
}

void loop() {
 l.loop()
}


```

Reply:
even like `l.triggerLightning()` or `l.sunrise()` maybe?

Reply:
Yeah, I was thinking that too.  Been a while since I did c++ so I was sticking to simple c right now.  I will probably go to this after I get more functionality.